place-work/v1alpha1/types.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	work "open-cluster-management.io/api/work/v1"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// PlaceManifestWorkSpec defines the desired state of PlaceManifestWork
+type PlaceManifestWorkSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// ManifestWorkSpec is the ManifestWorkSpec that will be used to generate a per-cluster ManifestWork
+	ManifestWorkSpec work.ManifestWorkSpec `json:"manifestWorkSpec"`
+
+	// PacementRef is the name of the Placement resource, from which a PlacementDecision will be found and used
+	// to distribute the ManifestWork
+	PlacementRef *corev1.LocalObjectReference `json:"placementRef,omitempty"`
```

Comment:
it is better to create our own placement ref instead of using corev1 one.

Reply:
Done.